### ISSUE TITLE
Bed Shape Not Saved

### ISSUE THREAD

**Issue by ShryuKt** [SoftFever/OrcaSlicer#83]
Hi, 

Just a bug I found.
Even if I select a Texture and Model for the Bed Shape, It is not saved.

Select
![image](https://user-images.githubusercontent.com/115975090/210151823-b062651c-f5db-44ec-91a7-da38f595c847.png)

OK and reopen
![image](https://user-images.githubusercontent.com/115975090/210151826-4cc5bf65-6c38-4d1e-80ad-717b9fd814f5.png)

And of course the texture and the model doesn't change.

Thanks
-- Shryû --

**Comment by Arakon** [comment]
This issue still occurs. Is there a workaround? I looked around in the config json files, but there's not even a mention of these settings that I could find.

**Comment by Icesythe7** [comment]
> This issue still occurs. Is there a workaround? I looked around in the config json files, but there's not even a mention of these settings that I could find.

Go to `C:\Users\Icesythe7\AppData\Roaming\OrcaSlicer\user\{random number here}\machine` open your printer.json file
insert on line 2 and 3
```
"bed_custom_model": "C://Users//pathtoyourstl//printbed.stl",
"bed_custom_texture": "C://Users//pathtoyourpng//Rook_Grid.png",
```

then save and restart
The error seems to be that when saving the texture/model it is omitting the "//" in the path thus making the path invalid and it gets removed.

**Comment by SoftFever** [comment]
Fixed in 1.6.2 beta
